android/shopify_buy_plugin/src/main/java/com/shopify/unity/buy/WebIntentFragment.java

```diff
@@ -0,0 +1,41 @@
+package com.shopify.unity.buy;
+
+import android.app.Fragment;
+import android.net.Uri;
+import android.support.customtabs.CustomTabsIntent;
+
+public class WebIntentFragment extends Fragment {
+
+    private boolean didShowIntent = false;
```

Comment:
Yes, when they return to the application we check to see if they have completed the checkout via the web intent. Checking whether they finished the web checkout on `TAB_HIDDEN` may be misleading, because they might end up navigating back to the web view.